api/docs/org.jboss.tools.rsp.schema/src/main/resources/client/typescript/src/protocol/generated/protocol.ts

```diff
@@ -3,7 +3,7 @@
  */
 export namespace Protocol {
     /* tslint:disable */
-    // Generated using typescript-generator version 2.2.413 on 2019-06-03 14:27:41.
+    // Generated using typescript-generator version 2.2.413 on 2019-06-05 13:54:35.
```

Comment:
This change should be removed from the PR for better clarity

Reply:
+1, but then I think that this new protocol.ts should not even be included in this PR that's about providing support for a new server.